### ISSUE TITLE
Remove batchnorm from vanilla SGD optimizer

### ISSUE THREAD

**Issue by jack-willturner** [jack-willturner/batchnorm-pruning#4]
Will also have to update the training function in `utils`
